packages/api/package.json

```diff
@@ -7,8 +7,8 @@
   "main": "dist/worker.js",
   "scripts": {
     "lt": "npm run lt:cluster & npm run lt:ipfs-proxy",
-    "lt:cluster": "npx localtunnel --port 9094 --subdomain \"$(whoami)-cluster-api-nft-storage\"",
-    "lt:ipfs-proxy": "npx localtunnel --port 9095 --subdomain \"$(whoami)-ipfs-proxy-api-nft-storage\"",
+    "lt:cluster": "npx localtunnel --port 9094 --subdomain ze-cluster-api-nft-storage",
+    "lt:ipfs-proxy": "npx localtunnel --port 9095 --subdomain ze-ipfs-proxy-api-nft-storage",
```

Comment:
Please revert. It's better to use `whoami` so this works for everyone!

Reply:
ah this slipped through! reverting!